### DISCUSSION TITLE
Github copilot with tool of student

### DISCUSSION THREAD

**Discussion by Luan-Reis** [community/community#66616] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I would just like to ask a question. I have access to github student and when it occurred to me I access github copilot for free. How long will I have access to this tool for free as a student? Is there an expiration date?

**Comment by yeikel** [comment, marked as answer]
As long as you're a student (and can prove it)

See https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot#pricing-for-github-copilot-for-individuals

**Reply by Luan-Reis** [reply]
So, it's indefinite, right? I read the article you sent and it doesn't specify a limit.

**Reply by yeikel** [reply]
That's what it seems like. As long as you're a student

**Reply by Luan-Reis** [reply]
Yes. I am. Thank you very much.
